apps/base-docs/docs/tools/node-providers.md

```diff
@@ -192,6 +192,15 @@ hide_table_of_contents: true
 
 ---
 
+## RockX
+[RockX](https://access.rockx.com) offers a global blockchain node network and developer tools for Web3 innovation. Start with our free [Base RPC] (https://access.rockx.com/product/base-blockchain-api-for-web3-builders) to access institutional-grade solutions.
```

Comment:
@agol586 

There's a missing space that's breaking the hyperlink for `Base RPC` could you please fix?